settings.gradle.kts

```diff
@@ -21,3 +21,4 @@ plugins {
 include("testmod")
 include("testmod1.12")
 include("testdepmod")
+include("testnoforgeenv")
```

Comment:
This test mod should go into one of the tests in `functionalTest`, not as a subfolder

Reply:
Shall I just assert the task `packagePatchedMc` being successful or shall I also assert for the non-existence of forge's files in the output jar?

Reply:
If it's not too complicated, test for any entry that begins with `net/minecraftforge` or something like this

Reply:
I believe the side notations reside in a package starting with `net/minecraftforge`, shall I special-case for those 2 classes?

Reply:
yeah